runtime/src/bank.rs

```diff
@@ -2099,7 +2100,9 @@ impl Bank {
             return;
         }
 
-        self.distribute_rent_to_validators(&self.vote_accounts(), rent_to_be_distributed);
+        let leftover =
+            self.distribute_rent_to_validators(&self.vote_accounts(), rent_to_be_distributed);
+        self.capitalization.fetch_sub(leftover, Ordering::Relaxed);
```

Comment:
Just to confirm, this is ok because `capitalization` is not a part of the bank hash right?

Reply:
> Just to confirm, this is ok because `capitalization` is not a part of the bank hash right?

Exactly. :)